packages/pigeon/lib/java_generator.dart

```diff
@@ -313,8 +303,37 @@ String _makeSetter(Field field) {
   return 'set$uppercased';
 }
 
-String? _javaTypeForDartType(String datatype) {
-  return _javaTypeForDartTypeMap[datatype];
+String _flattenTypeArguments(List<TypeArgument> args) {
```

Comment:
As with the Dart version, a comment would be helpful.

Reply:
Done.  (I hope to factor this out to generator tools at some point)